src/master/develop/management.md

```diff
@@ -250,11 +264,218 @@ If you're using another development language, please refer to the [immugw](/mast
 
 ::::
 
+### Database listing
+
+This example shows how to list existent databases using `DatabaseListV2` method.
+
+:::: tabs
+
+::: tab Go
+
+```go
+	client, err := immudb.NewClient()
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	ctx := context.Background()
+	
+	err = client.OpenSession(ctx, []byte(`immudb`), []byte(`immudb`), "defaultdb")
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	defer client.CloseSession(ctx)
+
+	res, err := client.DatabaseListV2(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, db := range res.Databases {
+		fmt.Printf("database: %s, loaded: %v\r\n", db.Name, db.Loaded)
+	}
+```
+:::
+
+::: tab Java
+```java
+	List<String> dbs = immuClient.databases();
+	// List of database names
+```
+:::
+
+::: tab Python
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+:::
+
+::: tab Node.js
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [Node.js sdk github project](https://github.com/codenotary/immudb-node/issues/new)
+:::
+
+::: tab .Net
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [.Net sdk github project](https://github.com/codenotary/immudb4dotnet/issues/new)
+:::
+
+::: tab Others
+If you're using another development language, please refer to the [immugw](/master/immugw/) option.
+:::
+
+::::
+
+### Database loading/unloading
+
+Databases can be dynamically loaded and unloaded without having to restart the server. After the database is unloaded, all its resources are released. Unloaded databases cannot be queried or written to, but their settings can still be changed.
+Upon startup, the immudb server will automatically load databases with the attribute `Autoload` set to true.
```

Comment:
I'd also add that the default for new databases is to autoload,

also I'd add that if for whatever reason the database can not be loaded upon startup, it will remain closed.